tests/resources/aria-at-test-run.mjs

```diff
@@ -155,34 +163,35 @@ export function instructionDocument(resultState, hooks) {
     },
     instructions: {
       header: {
-        header: `Testing task: ${resultState.info.description}`,
+        header: `Test: ${resultState.info.description}`,
         focus: resultState.currentUserAction === UserActionMap.LOAD_PAGE,
       },
       description: `${modePhrase} how ${resultState.config.at.name} behaves when performing task "${lastInstruction}"`,
       instructions: {
-        header: 'Test instructions',
+        header: 'Instructions',
         instructions: [
           [
-            `Restore default settings for ${resultState.config.at.name}. For help, read `,
+            `Configure ${resultState.config.at.name} with default settings. For help, read `,
```

Comment:
This line doesn't match the text I see in the issue https://github.com/w3c/aria-at/issues/1010 is there documentation for this change anywhere?

Reply:
That's text to match against any of the [`ats[].defaultConfigurationInstructionsHTML`](https://github.com/w3c/aria-at/blob/master/tests/support.json) values

> This line doesn't match the text I see in the issue https://github.com/w3c/aria-at/issues/1010

What text are you referencing there?

Reply:
Oh I think I misread it. Actually, it looks right.